src/Header.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+import styled from 'styled-components/macro'
+
+export default function Header() {
+  return <HeaderStyled> Mein Austauschjahr</HeaderStyled>
```

Comment:
Kannst du den Text 'Main Austauschjahr' auslagern in deine App.js?
Als children könntest du das durchreichen, dann weiß der Header nichts von irgendwelchem text.